src/js/actions/FriendActions.js

```diff
@@ -39,7 +39,8 @@ module.exports = {
         email_addresses_raw: email_addresses,
         invitation_message: invitation_message,
         ballot_link: ballot_link,
-        sender_email_address: sender_email_address
+        sender_email_address: sender_email_address,
```

Comment:
ES6 allows for shorthand key value. Rather than:

`sender_email_address: sender_email_address`

you can do:

`sender_email_address`

That goes for all of the keys listed here. Might be best to name the args passed in the same as the keys so you can reduce this to just having to use the key name.

Reply:
Great suggestion @pertrai1! We will start transitioning to this new shorthand.